### PR TITLE
Use CuPy v7 in CI

### PR DESCRIPTION
**Pull request by emcastillo**

### CODE REVIEW

**Comment by emcastillo** [comment]
Jenkins, test this please

**Comment by emcastillo** [comment]
Jenkins, test this please

**Comment by emcastillo** [comment]
Jenkins, test this please

**Comment by emcastillo** [comment]
pfnCI, test this please

**Comment by emcastillo** [comment]
pfnCI, test this please

**Comment by emcastillo** [comment]
Jenkins, test this please

**Comment by emcastillo** [comment]
Jenkins, test this please

**Comment by emcastillo** [comment]
Jenkins, test this please

**Comment by emcastillo** [comment]
Jenkins, test this please

**Comment by emcastillo** [comment]
Jenkins, test this please

**Comment by emcastillo** [comment]
Jenkins, test this please

**Comment by emcastillo** [comment]
Jenkins, test this please

**Comment by emcastillo** [comment]
Jenkins, test this please

File: .pfnci/hint.pbtxt [hunk 1]
```diff
@@ -10,6 +10,7 @@ rules { name: "chainer_tests/functions_tests/loss_tests/test_softmax_cross_entro
 rules { name: "chainer_tests/functions_tests/math_tests/test_sparse_matmul.py" xdist: 4 deadline: 240 }
 rules { name: "chainer_tests/functions_tests/normalization_tests/test_batch_normalization.py" xdist: 4 deadline: 240 }
 rules { name: "chainer_tests/functions_tests/pooling_tests/test_max_pooling_nd.py" xdist: 4 deadline: 240 }
+rules { name: "chainer_tests/functions_tests/rnn_tests/test_function_slstm.py" xdist: 4 deadline: 240}
```
Comment (emcastillo) [inline .pfnci/hint.pbtxt:13]: This solves lstm deadline

**Comment by emcastillo** [comment]
Jenkins, test this please

**Comment by emcastillo** [comment]
Jenkins, test this please

**Comment by emcastillo** [comment]
Jenkins, test this please

**Comment by emcastillo** [comment]
Jenkins, test this please

**Comment by emcastillo** [comment]
Jenkins, test this please

**Comment by emcastillo** [comment]
Jenkins, test this please

**Comment by kmaehashi** [comment]
pfnCI, test this please.

**Comment by kmaehashi** [comment]
LGTM!

**Pull request merged**